cwf/cloud/go/services/analytics/analytics/main.go

```diff
@@ -14,252 +14,9 @@
 package main
 
 import (
-	"flag"
-	"magma/cwf/cloud/go/services/analytics/calculations"
-	"strconv"
-	"time"
-
-	"magma/cwf/cloud/go/cwf"
-	"magma/cwf/cloud/go/services/analytics"
-	"magma/orc8r/cloud/go/orc8r"
-	"magma/orc8r/cloud/go/service"
-	"magma/orc8r/cloud/go/services/metricsd"
-	"magma/orc8r/lib/go/metrics"
-	"magma/orc8r/lib/go/service/config"
-
-	"github.com/golang/glog"
-	promAPI "github.com/prometheus/client_golang/api"
-	"github.com/prometheus/client_golang/api/prometheus/v1"
-	"github.com/prometheus/client_golang/prometheus"
-)
-
-const (
-	ServiceName = "ANALYTICS"
-
-	activeUsersMetricName           = "active_users_over_time"
-	userThroughputMetricName        = "user_throughput"
-	userConsumptionMetricName       = "user_consumption"
-	userConsumptionHourlyMetricName = "user_consumption_hourly"
-	apThroughputMetricName          = "throughput_per_ap"
-	authenticationsMetricName       = "authentications_over_time"
-
-	defaultAnalysisSchedule = "0 */12 * * *" // Every 12 hours
-)
-
-var (
-	// Map from number of days to query to size the step should be to get best granularity
-	// without causes prometheus to reject the query for having too many datapoints
-	daysToQueryStepSize = map[int]time.Duration{1: 15 * time.Second, 7: time.Minute, 30: 5 * time.Minute}
-
-	daysToCalculate = []int{1, 7, 30}
+	lteAnalytics "magma/lte/cloud/go/services/analytics"
```

Comment:
1. keep the main.go in the lte module

2. if you want to turn the analytics service into a generic metrics postprocessing thing, think about how to make it pluggable. because we don't want to have CWF analytic computations in the LTE codebase, for example. @hcgatewood  or @mpgermano might have some ideas for you r.e. service mesh registration and so forth

Reply:
Hey @xjtian , to provide some context here, I want to expand this LTE piece to collect more state/config from orc8r. I want the functionality in CWF to really remain as is and add more functionality to LTE. Based on your previous comments and Scott's comments I removed the CWF code and moved most to LTE and just retained main. We have main.go in CWF and LTE module now. I am confused on what to do here.  Also on your point 2, we don't have any CWF specific analytics computations in LTE codebase here. It is just similar metrics being used by both LTE and CWF modules.